cmd/wallet-web/src/pages/mixins/common/helper.js

```diff
@@ -116,8 +116,11 @@ export function getCredentialDisplayData(vc, manifest) {
   // fallback credential will use vc credentialSubject as properties
   const properties =
     Object.values(manifest.properties).length === 0
-      ? vc.credentialSubject
-      : Object.entries(manifest.properties).map(_readProperty);
+      ? Object.entries(vc.credentialSubject).map(([label, value]) => ({
+          label,
+          value,
+        }))
+      : Object.values(manifest.properties).map(_readProperty);
```

Comment:
We don't need this function anymore

Reply:
this is required you posted comment on old code